### ISSUE TITLE
The folding stops at the first unfoldable line matching the pattern

### ISSUE THREAD

**Issue by serby2000** [baincd/vscode-custom-auto-fold#1]
### Subject of the Issue
When a line matches the **linePattern** regex pattern but it can't be folded the next lines that match the pattern are not folded anymore.

### Environment
* VSCode v1.68.1
* Custom Auto Fold v1.1.0

#### settings.json
```json
"custom-auto-fold.rules": [
        {
            "fileGlob": "**/*.txt",
            "linePattern": "^FF.*",
            "firstMatchOnly": false
        },
]
```
#### textfold.txt
```txt
FF
    haha
    bubu
FF the line that stops the folding
FF
    haha
    bubu
FF
    john
```

### Expected Behavior

To fold all the lines that match the pattern and can be folded

### Actual Behavior

The folding stops at the first unfoldable line matching the pattern

**Comment by baincd** [comment]
@serby2000 Thank you for the excellent write up!  I'll take a look into this when I have some time

**Comment by baincd** [comment]
Fix in v1.1.1
